File: my-folder/0125-valid-palindrome/solution.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        contString = ''

        for i in range(len(s)):
            if s[i].isalnum():
                contString += s[i].lower()
        
        l, r = 0, len(contString)-1
        while l <= r:
            if contString[l] != contString[r]: return False
            l += 1
            r -= 1
        return True

File: my-folder/0125-valid-palindrome/solution.cs
public class Solution {
    public bool IsPalindrome(string s) {
        string result = "";
        foreach(char c in s) {
            if (char.IsLetterOrDigit(c)) {
                result += char.ToLower(c);
            }
        }

        return (result == new string(result.Reverse().ToArray()));
    }
}
